app/controllers/miq_policy_controller/alert_profiles.rb

```diff
@@ -168,80 +168,50 @@ def process_alert_profiles(alert_profiles, task)
     process_elements(alert_profiles, MiqAlertSet, task)
   end
 
-  # Gather up the object ids based on the assignment selections
-  def alert_profile_get_assign_to_objects
-    objs = []
+  def alert_profile_get_assign_to_objects_empty?
+    empty = true
```

Comment:
early return to avoid nesting:

```
def  alert_profile_get_assign_to_empty?
  return true if @assign[:new][:assign_to].blank?
  return true if @assign[:new][:assign_to] == "enterprise"
  return true if @assign[:new][:assign_to].ends_with?("-tags") && @assign[:new][:cat].blank?
end
```

plus clearly shows the relation between the test and the return value